app/src/main/java/com/anzelmas/quizzando/QuizQuestionActivity.kt

```diff
@@ -14,6 +15,145 @@ import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
 import kotlinx.android.synthetic.main.activity_quiz_question.*
 
+class QuizQuestionActivity : AppCompatActivity(), View.OnClickListener {
+
+
+    private var currentPosition: Int = 1
+    private var questionList: ArrayList<Question>? = null
```

Comment:
`private var questionList: ArrayList<Question> = Constants.getQuestions()` should work, I believe